core/kazoo_apps/src/kapps_maintenance.erl

```diff
@@ -174,10 +175,10 @@ refresh(?KZ_OAUTH_DB) ->
 refresh(?KZ_WEBHOOKS_DB) ->
     kz_datamgr:db_create(?KZ_WEBHOOKS_DB),
     kz_datamgr:revise_doc_from_file(?KZ_WEBHOOKS_DB, 'crossbar', <<"views/webhooks.json">>),
-    webhooks_maintenance:reset_webhooks_list();
+    kazoo_bindings:map(binding(refresh_webhooks), []);
```

Comment:
I think the kz_datamgr lines above should be part of the `webhooks_maintenance` callback. `refresh/1` should use the Db to build the `refresh_binding/1` for mapping. That's my initial thought, anyway.